app/controllers/collections_controller.rb

```diff
@@ -5,7 +5,7 @@ class CollectionsController < ApplicationController
   before_action :set_current_user_collection, only: %i[update edit destroy]
 
   def index
-    @collections = Collection.all
+    @collections = Collection.paginate(:page => params[:page], per_page: Rails.configuration.x.WillPaginate.per_page)
```

Comment:
Like the `x` 👍 
Any reason you departed from snake case for the `WillPaginate`? 
Why not `will_paginate`?
Or even just `Rails.configuration.x.results_per_page`?
Does the will paginate gem add this to the config for you?

Reply:
Moved comment to other PR #44